### ISSUE TITLE
has the 5V pin from Daikin AC enough mA to power the ESP?

### ISSUE THREAD

**Issue by rrelande** [msx80/FakeA41#1]
Thanks for a fantastic project!

As we know, Daikin exposes 5V and 12V signals on the S21. 
we know that the Daikin original A41 uses 12V and it is logical to use 12V too for the power supply of the ESP. 

However, it would be so much easier to use directly the 5V pin to supply power to the ESP. 

Has anybody tried to feed 5V to the ESP ? Is the AC able to supply enough mA at this voltage from the S21?

**Comment by msx80** [comment]
Hi there, thanks :) I haven't tried to use the 5v directly, i don't remember why actually 😅. Maybe it wasn't available on all devices? Or it was safer to use what the A41 used?

Anyway, the esp should consume less than 200mA while transmitting and less than 100mA while listening. Pretty low requirements, i'd be surprised if the port can't supply it. Perhaps with a capacitor to offset peaks during transmission. You'd still need some kind of level shifter but could remove the voltage regulator!

**Comment by rrelande** [comment]
after testing with daikin unit and observing on thermal imaging, I could see that the daikin power regulator was heating more than usual when ESP is power on 5V. 
I checked the specs of the power regulator, it is made for 500 mA. 
Furthermore the daikin power regulator does not have the heatsink which further derates the maximum power supply capability of the Daikin 5V regulator. 

clearly, unless you use the ESP without radio, you can't use the 5V but need to use the 12V power supply

**Comment by msx80** [comment]
Thanks for your research @rrelande! In theory 500mA should be enought for a fully wake esp32, perhaps there's some other stuff attached to that power regulator..

**Comment by Amtho** [comment]
I find the 500ma a little bit on the edge becuase we dont know what is else powered with the internal regulator.
Therefore I will use the 12V with a stepdown to power the esp and will only use the 5V for the levelshifter with an ADUM1201.

**Comment by rrelande** [comment]
indeed, the regulator is on the board for a reason and it has been selected by Daikin engineers to exactly support the board functions. There is probably some margin, but hardly the full 500 mA can be used for ESP on permanent basis.

**Comment by msx80** [comment]
I've added a note about the issue on the Readme, i'll close this issue later if you have nothing to add.

Thanks again everyone
